lib/management.js

```diff
@@ -0,0 +1,430 @@
+const async = require('async');
+const forge = require('node-forge');
+const arsenal = require('arsenal');
+const config = require('../conf/Config');
+const AWS = require('aws-sdk');
+const werelogs = require('werelogs');
+const bucketclient = require('bucketclient');
+
+werelogs.configure({
+    level: config.log.logLevel,
+    dump: config.log.dumpLevel,
+});
+const logger = new werelogs.Logger('mdManagement');
+
+const serviceBucket = 'PENSIEVE';
+const refreshInterval = 5000;
+
+const Metadata = arsenal.storage.metadata.MetadataWrapper;
+let metadata;
+
+const params = {
```

Comment:
I think you'll need to condition this also to `REMOVE_MANAGEMENT_DISABLE` var, so that `config.queuePopulator.dmd` is not accessed if the var is set.

Reply:
Or maybe declare it inside `initManagement`?